errata_tool/erratum.py

```diff
@@ -888,6 +888,28 @@ def commit(self):
             self.refresh()
         return ret
 
+    def push(self, target='stage'):
+        # Accept 'stage', 'live', or a set of specific options
+        # (See ET documentation)
+        if self.errata_id == 0:
+            return False
+
+        url = '/api/v1/erratum/' + str(self.errata_id) + '/push'
+        if isinstance(target, str) or isinstance(target, unicode):
+            if target != 'stage' and target != 'live':
+                raise ValueError('Wrong value for target: expected ' +
+                                 '\'stage\', \'live\', or a list')
+
+            r = self._post(url + '?defaults=' + str(target))
+            self._processResponse(r)
+            return True
+        if not isinstance(target, list):
+            raise ValueError('Wrong value for target: expected ' +
+                             '\'stage\', \'live\', or a list')
+        r = self._post(url, data=target)
+        self._processResponse(r)
```

Comment:
Is there any point to returning the response here? (does `r` have anything useful?)

Reply:
Raises exceptions when things blow up, same as other code paths?

Reply:
I misread what you asked - yes, returning the JSON response makes total sense.

Reply:
Looks great!